dev/com.ibm.ws.logging/src/com/ibm/ws/logging/internal/impl/BaseTraceService.java

```diff
@@ -491,6 +485,8 @@ public static void applyJsonFields(String value, Boolean omitJsonFields) {
             return;
         }
 
+        CollectorJsonHelpers.updateFieldMappings();
```

Comment:
this has to be after the field mappings are updated (ie. after line 596), otherwise you have a race condition.